packages/hapi/modules/plugin/plugin.js

```diff
@@ -81,7 +83,16 @@ const createPlugin = ({ options } = {}) => {
       server.decorate('server', 'Prometheus', Prometheus);
       server.decorate('server', 'recordRequest', recordRequest);
 
-      return onDone && onDone();
+      server.route({
```

Comment:
my thought is why need to have another http server running to provide metrics API, can easily disable the public access in nginx or ingress. for plugin, it should be injectable/configurable and less code. by adding a default route and configurable, this will bring less code change to the application and using the same port, which prometheus scrap normally target.

Reply:
You have another server if you just use `@promster/server`. To be clearer: I do not plan to have another approach on that here.